tests/io_utils.go

```diff
@@ -75,12 +75,14 @@ func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (stdout stri
 	return stdout, nil
 }
 
+// The tests using the function CreateErrorDisk need to be run serially as it relies on the kernel scsi_debug module
```

Comment:
We can actually assert that we run serially.

Reply:
@alicefr fyi

Reply:
@xpivarc merged too fast, we can add it in a second PR. However, I'm not quite sure, don't we need the serial decorator on the test description?

Reply:
Yes, you do. What I am suggesting is that you enforce that this function is only called from Serial tests.